### PR TITLE
Added Symphony and Harmony as a presence

### PR DESCRIPTION
**Pull request by xxdjbobbyxx**
I have checked everything and it all works and I have added:
Symphony
Harmony as a radio presence

### CODE REVIEW

**Review by Bas950 — CHANGES_REQUESTED** [review]
Undo change in package.json
Fix codacy errors
Version 1.0.0 on release

**Comment by xxdjbobbyxx** [comment]
I have changed the following ages ago but for some reason git push never did anything so I have manually changed them and fixed the errors

File: websites/H/Harmony/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  },
+  "url": "harmonyradio.pw",
+  "description": {
+    "en": "Harmony Radio is a online radio station bringing you music 24/7"
+  },
+  "service": "Harmony",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/I8s5X6q.jpg",
+  "thumbnail": "https://i.imgur.com/CAMa2EI.png",
+  "color": "#46356e",
```
Comment (Bas950) [inline websites/H/Harmony/dist/metadata.json:15]: why is the thumbnail blurred?

File: websites/H/Harmony/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "author": {
+    "name": "xxdjbobbyxx",
+    "id": "456601190671843328"
+  },
+  "url": "harmonyradio.pw",
+  "description": {
+    "en": "Harmony Radio is a online radio station bringing you music 24/7"
+  },
+  "service": "Harmony",
```
Comment (Bas950) [inline websites/H/Harmony/dist/metadata.json:11]: change service name and folder name to "Harmony Radio"

**Review by Bas950 — CHANGES_REQUESTED** [review]
Don't upload the JS files.
Upload proof of it working.

**Comment by xxdjbobbyxx** [comment]
Everything has been fixed and as you can see I have made sure everything works!
image
image

**Comment by ririxichan** [comment]
Harmony radio is not 2months old. Symphony one is on free domain. We do not accept these websites. Feel free to open pull request at 19.11.2020 for harmony radio but not for symphony.

**Comment by DefectivenessYT** [comment]
sorry to butt in, but these sites are totally not 2 months old as stated in the contributor rules. both of these were made recently and one of them is on a free domain

**Comment by DefectivenessYT** [comment]
oh ririxichan you beat me to it

**Pull request closed without merging**